### PR TITLE
Update workshop page

### PR DESCRIPTION
**Pull request by carme-hp**
- add important dates section
- change what belongs to a note

### CODE REVIEW

**Comment by carme-hp** [comment]
image

File: pages/community/precice-workshop-2024.md [hunk 1]
```diff
@@ -11,28 +11,28 @@ redirect_from: /preCICE2024/
...
 
-The workshop will start with a hands-on training course on September 24-25. The [course](https://precice.org/community-training.html) is suited for both beginners and current preCICE users, since advanced topics will also be covered. The main part of the workshop will take place on September 25-27. A detailed program for the workshop will be available soon.
+The workshop will start with a hands-on training course on September 24-25. The [course](https://precice.org/community-training.html) is suited for both beginners and current preCICE users, since advanced topics will also be covered. The main part of the workshop will take place on September 25-27.
 
 Keep watching this space for updates on registration, the workshop program, and more.
 
+### Important dates
+
+| Abstract submission deadline | **May 31**  |
+| Early-bird registration deadline      | **June 30**  |
```
Comment (IshaanDesai) [inline pages/community/precice-workshop-2024.md]:
```suggestion
| Early-bird registration deadline      | **July 31**  |
```

**Review by IshaanDesai — APPROVED** [review]
Just one correction. The rest looks good :+1:

**Comment by carme-hp** [comment]
image

**Pull request merged**